src/main/java/serverutils/ServerUtilitiesConfig.java

```diff
@@ -544,6 +544,14 @@ public boolean log(EntityPlayerMP player) {
         @Config.DefaultBoolean(false)
         public boolean show_playtime;
 
+        @Config.Comment("Enabled Player Sleeping Percentage to skip night. Use the gamerule playersSleepingPercentage to set the percentage.")
+        @Config.DefaultBoolean(false)
```

Comment:
I vote that this gets enabled by default. You can hit it with a `@Config.ModDetectedDefault` to change the default to off if the EFR coremod is present.

Reply:
Nevermind digging into GTNHLib it's just the fully qualified class for the IFMLLoadingPlugin. This should be good now.